DigitalLearningSolutions.Web/ViewModels/TrackingSystem/CourseSetup/CourseStatisticsViewModelFilterOptions.cs

```diff
@@ -57,6 +52,16 @@ IEnumerable<string> topics
                     CourseHasAdminFieldOptions
                 ),
             };
+
+            return categories.Any()
+                ? filterOptions.Prepend(
+                    new FilterModel(
+                        nameof(CourseStatistics.CategoryName),
+                        "Category",
+                        GetCategoryOptions(categories)
+                    )
+                )
+                : filterOptions;

```

Comment:
I'm getting Possible multiple enumeration warnings in Rider for `categories` here